src/main/java/rx/util/async/Async.java

```diff
@@ -15,19 +15,60 @@
  */
 package rx.util.async;
 
-import rx.*;
+import java.util.concurrent.Callable;
+import java.util.concurrent.Future;
+import java.util.concurrent.FutureTask;
+
+import rx.Observable;
+import rx.Observer;
+import rx.Scheduler;
 import rx.Scheduler.Worker;
-import rx.functions.*;
+import rx.Subscriber;
```

Comment:
Could you make sure these remain as * to so the header of the class won't show as merge conflict in case there are multiple PRs?